### PR TITLE
website finalized

### PR DESCRIPTION
**Pull request by dakshj48**

### CODE REVIEW

File: audioCon.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 
 def convert(inputfile, outputfile, period):
+    if period < 0:
```
Comment (sharath) [inline audioCon.py]: I don't think it makes sense to do this. Just do input verification instead.

File: templates/upload.html [hunk 2]
```diff
@@ -1,14 +1,38 @@
...
+                </p>
+                <script>
+                    function converting() {
+                        document.getElementById("submit").style.visibility = 'hidden';
+                        document.getElementById("loading").style.visibility = 'visible';
+                    }
+                </script>
+            </form>
+        </div>
+        <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```
Comment (sharath) [inline templates/upload.html]: Can you defer these and move them into the head?

File: static/upload.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
+body {
+    background-color: #0B0C10;
```
Comment (sharath) [inline static/upload.css]: I don't know about this color...

File: audioCon.py (hunk 1 above)
Comment (dakshj48) [reply audioCon.py]: Do you mean negative values are acceptable?

File: static/upload.css (hunk 3 above)
Comment (dakshj48) [reply static/upload.css]: I wanted to make a dark themed website. Any recommendations?

**Pull request merged**